ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/sync/ChainHeadTracker.java

```diff
@@ -83,8 +83,10 @@ public void onPeerConnected(final EthPeer peer) {
                 peer.chainState().update(chainHeadHeader);
                 trailingPeerLimiter.enforceTrailingPeerLimit();
               } else {
-                LOG.debug(
-                    "Failed to retrieve chain head information.  Disconnecting {}. ", peer, error);
+                LOG.debug("Failed to retrieve chain head information.  Disconnecting {}. ", peer);
+                if (error != null) {
+                  LOG.debug("Error reporting while retrieving chain head information", error);
+                }
```

Comment:
Personally I'd just use:
```
LOG.debug("Failed to retrieve chain head information. Disconnecting " + peer, error);
```
the cost of string concatenation doesn't matter for this fairly rare case and then everything is together and nice and simple.